### ISSUE TITLE
About kitti test result

### ISSUE THREAD

**Issue by muomi** [yifanzhang713/IA-SSD#13]
Hi, yifan! Thanks for you wounderful work. I eval your model on kitti val dataset and get the same result as your repo. However, when i submit the test result on kitti website. I can not get the result in your paper. 
2022-04-21 00:46:25,841   INFO  *************** Performance of EPOCH no_number *****************
2022-04-21 00:46:25,841   INFO  Generate label finished(sec_per_example: 0.0177 second).
2022-04-21 00:46:25,841   INFO  recall_roi_0.3: 0.000000
2022-04-21 00:46:25,841   INFO  recall_rcnn_0.3: 0.927099
2022-04-21 00:46:25,842   INFO  recall_roi_0.5: 0.000000
2022-04-21 00:46:25,842   INFO  recall_rcnn_0.5: 0.884041
2022-04-21 00:46:25,842   INFO  recall_roi_0.7: 0.000000
2022-04-21 00:46:25,842   INFO  recall_rcnn_0.7: 0.663060
2022-04-21 00:46:25,848   INFO  Average predicted number of objects(3769 samples): 7.721
2022-04-21 00:46:57,396   INFO  Car AP@0.70, 0.70, 0.70:
bbox AP:96.2779, 90.1569, 89.5057
bev  AP:90.3980, 88.8603, 86.9664
3d   AP:89.3976, 79.5625, 78.4435
aos  AP:96.25, 90.10, 89.35
Car AP_R40@0.70, 0.70, 0.70:
bbox AP:97.9059, 95.3214, 92.7594
bev  AP:94.7520, 91.3533, 88.8415
3d   AP:91.7990, 83.3737, 80.3455
aos  AP:97.88, 95.23, 92.57
Car AP@0.70, 0.50, 0.50:
bbox AP:96.2779, 90.1569, 89.5057
bev  AP:96.3642, 90.2276, 89.7638
3d   AP:96.3233, 90.2013, 89.6963
aos  AP:96.25, 90.10, 89.35
Car AP_R40@0.70, 0.50, 0.50:
bbox AP:97.9059, 95.3214, 92.7594
bev  AP:97.9552, 95.6085, 95.0305
3d   AP:97.9302, 95.5291, 94.8719
aos  AP:97.88, 95.23, 92.57
Pedestrian AP@0.50, 0.50, 0.50:
bbox AP:72.7975, 70.9381, 67.4165
bev  AP:66.7870, 61.3725, 57.6589
3d   AP:60.8069, 58.3171, 52.2580
aos  AP:68.50, 65.93, 62.32
Pedestrian AP_R40@0.50, 0.50, 0.50:
bbox AP:74.5996, 70.7681, 67.1326
bev  AP:66.4324, 61.7818, 56.7088
3d   AP:61.6833, 56.9830, 51.8147
aos  AP:69.51, 65.15, 61.29
Pedestrian AP@0.50, 0.25, 0.25:
bbox AP:72.7975, 70.9381, 67.4165
bev  AP:81.5397, 78.5586, 73.2419
3d   AP:81.4691, 78.5135, 73.1643
aos  AP:68.50, 65.93, 62.32
Pedestrian AP_R40@0.50, 0.25, 0.25:
bbox AP:74.5996, 70.7681, 67.1326
bev  AP:82.5906, 79.6797, 75.2987
3d   AP:82.5212, 79.6466, 75.1587
aos  AP:69.51, 65.15, 61.29
Cyclist AP@0.50, 0.50, 0.50:
bbox AP:95.5281, 78.1780, 76.5017
bev  AP:93.1435, 74.5972, 72.0342
3d   AP:85.1286, 71.1990, 68.6697
aos  AP:95.43, 77.85, 76.05
Cyclist AP_R40@0.50, 0.50, 0.50:
bbox AP:96.6252, 81.1220, 78.3019
bev  AP:94.3767, 75.5814, 72.4732
3d   AP:89.8377, 71.4524, 68.1976
aos  AP:96.52, 80.74, 77.81
Cyclist AP@0.50, 0.25, 0.25:
bbox AP:95.5281, 78.1780, 76.5017
bev  AP:94.4477, 79.1653, 74.1242
3d   AP:94.4477, 79.1653, 74.1242
aos  AP:95.43, 77.85, 76.05
Cyclist AP_R40@0.50, 0.25, 0.25:
bbox AP:96.6252, 81.1220, 78.3019
bev  AP:95.6348, 79.5295, 75.5297
3d   AP:95.6348, 79.5295, 75.5297
aos  AP:96.52, 80.74, 77.81

![image](https://user-images.githubusercontent.com/42430631/164288511-62b0a101-f3b4-44b1-8ba2-65ed4bbb61d3.png)

**Comment by yifanzhang713** [comment]
Hi @muomi ，thank you for following our work！

The weight file here is only trained on the training split (3712 frames) and then evalutated on validation split (3769 frames) to quantify the performance of the detectors. If submitting on the KITTI test server, the entire training set (7481 frames) with elaborate tuning should be taken into consideration in additional training stage.

**Comment by jwma0725** [comment]
Hi @yifanzhang713,  thank you for your work!
I want to know which model you choose to test and submit to the test server if you use all the training data.

**Comment by twn29004** [comment]
> Hi @muomi，thank you for following our work！
same question. Did you solve it?
> The weight file here is only trained on the training split (3712 frames) and then evalutated on validation split (3769 frames) to quantify the performance of the detectors. If submitting on the KITTI test server, the entire training set (7481 frames) with elaborate tuning should be taken into consideration in additional training stage.

**Comment by muomi** [comment]
> > Hi @muomi，thank you for following our work！
> > same question. Did you solve it?
> > The weight file here is only trained on the training split (3712 frames) and then evalutated on validation split (3769 frames) to quantify the performance of the detectors. If submitting on the KITTI test server, the entire training set (7481 frames) with elaborate tuning should be taken into consideration in additional training stage.

No...

**Comment by twn29004** [comment]
> Hello, can you give me a detection code submitted to Kitti on openpcdet? I just learned this knowledge and wrote a code myself, but the submitted result is very low. I hope I can refer to it

you can use the command `python tesst.py --cfg ./your_cfg_path.yaml --save_to_file `

**Comment by libingDY** [comment]
Thank you, but you understand me wrong. I want a tag file that can generate test sets to submit the code to Kitti.

**Comment by Tongjiaxun** [comment]
> 

same question. Did you solve it ?
